EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -68,7 +74,8 @@ public function __construct(
         $tokenParameterName,
         $singleUse,
         RefreshTokenGeneratorInterface $refreshTokenGenerator,
-        ExtractorInterface $extractor
+        ExtractorInterface $extractor,
+        $cookie
```

Comment:
```suggestion
        array $cookieSettings
```

Typehint the parameter and rename the parameter to make it a bit more clear what this array actually is (it's not an array with the cookie data within it, it's an array with the settings to configure a response cookie).